src/lib/unifyPageContent.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { renderToString } from 'react-dom/server';
+
+export const unifyPageContent = ({
+    css,
+    js,
+    html,
+}: {
+    css: string;
+    js: string;
+    html: string;
+}) =>
+    renderToString(
+        <html>
```

Comment:
Think we're missing a few bits from https://github.com/guardian/frontend/blob/master/common/app/views/fragments/atoms/interactive.scala.html like the doctype and meta. Looks like we also need to load interactive fonts in here.

Basically this file should be equivalent to that existing one.